scripts/policy/frameworks/management/controller/config.zeek

```diff
@@ -86,7 +86,7 @@ function network_info(): Broker::NetworkInfo
 	else if ( Management::default_address != "" )
 		ni$address = Management::default_address;
 	else
-		ni$address = "127.0.0.1";
+		ni$address = "0.0.0.0";
```

Comment:
Torn if the same applies here: when on a single-system cluster, make the default local? Controller is different of course because the client may run elsewhere, but still. Either way, would be helpful to show people how to change this setting, like with a short example in the docs.

Reply:
Yes, agree here too. I'll cover it.